src/Items/ItemBow.h

```diff
@@ -69,17 +69,12 @@ class cItemBowHandler :
 		}
 
 		// Create the arrow entity:
-		cArrowEntity * Arrow = new cArrowEntity(*a_Player, Force * 2);
-		if (Arrow == nullptr)
+		auto ArrowPtr = cpp14::make_unique<cArrowEntity>(*a_Player, Force * 2);
+		auto Arrow = ArrowPtr.get();
+		if (!Arrow->Initialize(std::move(ArrowPtr), *a_Player->GetWorld()))
```

Comment:
> but then I realized, what would get called first, the std::move or the indirection?

\*something about worktycho and standards\*

> We need to come up with a better interface for initialization (after this PR is merged).

Indeed. I would suggest that Initialize does not try to call AddEntity; whomever created the entity should. Also, merge Initialize into the constructor.